pypeit/wavecalib.py

```diff
@@ -523,6 +526,26 @@ def build_wv_calib(self, arccen, method, skip_QA=False):
                                              nonlinear_counts=self.nonlinear_counts,
                                              nsnippet=self.par['nsnippet'])
                                              #debug=True, debug_reid=True, debug_xcorr=True)
+        elif self.par['method'] == 'echelle':
```

Comment:
Should we instead put in a fault for now?  I.e., we don't want people using this until it's fully integrated.

Reply:
added.